bench_pytorch/requirements.txt

```diff
@@ -1,6 +1,6 @@
-transformers==4.34.1
-torch==2.0.0
-accelerate
+transformers==4.39.3
+torch==2.2.2
+accelerate==0.28.0
 sentencepiece
```

Comment:
add also sentencepiece version

Reply:
Ahh, this one got missed, yes lemme do that

Reply:
This is quite strange, because when I go the [file](https://github.com/premAI-io/benchmarks/blob/hf-mistral/bench_pytorch/requirements.txt) under `hf-mistral` branch, I see all the dependencies pinned with versions. 

https://github.com/premAI-io/benchmarks/blob/9eb30fe7f961ae93f6f6a96911b115a8f84f8e92/bench_pytorch/requirements.txt#L1-L6

Reply:
Done